packages/poaps/docs/PoapsClient.md

```diff
@@ -0,0 +1,183 @@
+# PoapsClient
+
+## Description
+
+`PoapsClient` is a class representing a client for interacting with POAPs .
+
+## Constructor
+
+```typescript
+constructor(
+  private compassProvider: CompassProvider,
+  private tokensApiProvider: TokensApiProvider
+)
+```
+
+- `compassProvider` (`CompassProvider`): The provider for the POAP compass API.
+- `tokensApiProvider` (`TokensApiProvider`): The provider for the Tokens API.
+
+## Methods
+
+### `fetch`
+
+```typescript
+async fetch(input: FetchPoapsInput): Promise<PaginatedResult<POAP>>
+```
+
+Fetches a list of POAP tokens based on the given input criteria.
+
+#### Parameters
+
+- `input` (`FetchPoapsInput`): Criteria for fetching POAP tokens.
+
+#### Returns
+
+A promise that resolves to a paginated list of POAP tokens.
```

Comment:
Link to paginated object showing how it returns data and to the POAP object to see what is returned.